frontend/public/components/service-binding.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+// eslint-disable-next-line no-unused-vars
+import { K8sResourceKindReference } from '../module/k8s';
+import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
+import { Cog, SectionHeading, navFactory, ResourceCog, ResourceLink, ResourceSummary, statusMessage, StatusWithIcon } from './utils';
+import { ResourceEventStream } from './events';
+import { Conditions } from './conditions';
+import { Parameters } from './parameters';
+
+const ServiceBindingsReference: K8sResourceKindReference = 'ServiceBinding';
+
+const { common } = Cog.factory;
+
+const menuActions = [
+  ...common,
+];
+
+const ServiceBindingDetails: React.SFC<ServiceBindingDetailsProps> = ({obj: sb}) => {
+  const sbStatusMessage = statusMessage(sb);
+  const sbParameters = _.get(sb, 'status.externalProperties.parameters', {});
+
+  return <React.Fragment>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Service Binding Overview" />
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={sb} showPodSelector={false} showNodeSelector={false} />
+        </div>
+        <div className="col-sm-6">
+          <dl className="co-m-pane__details">
+            <dt>Service Instance</dt>
+            <dd><ResourceLink kind="ServiceInstance" name={sb.spec.instanceRef.name} title={sb.spec.instanceRef.name} namespace={sb.metadata.namespace} /></dd>
+            <dt>Secret</dt>
+            <dd><ResourceLink kind="Secret" name={sb.spec.secretName} title={sb.spec.secretName} namespace={sb.metadata.namespace} /></dd>
+            <dt>Status</dt>
+            <dd><StatusWithIcon obj={sb} /></dd>
+            {sbStatusMessage && <dt>Status Message</dt>}
+            {sbStatusMessage && <dd>{sbStatusMessage}</dd>}
```

Comment:
I wonder if we need this since we already have the conditions.

Reply:
I wondered the same thing, but we'd included it for service instances, so I put it in here.  Imma remove it from both.